components/Card.js

```diff
@@ -84,4 +84,49 @@ CardAddress.propTypes = {
   label: PropTypes.string,
 };
 
+export function PositionRequirements({ label, children }) {
+  return (
+    <div className="p-2 shaded-text" style={{ marginTop: 'auto' }}>
+      <span className="font-semibold">{label}</span>
+      <ul className="list-disc text-left pl-6">
+        {children.map(function (listItem, index) {
```

Comment:
could be simplified to:
```javascript
children.map((listItem, index) => <li key={index}>{listItem}</li>
```

Reply:
Yep! Good point! That's because at first I wasn't using `children` and I forgot to update it. Thanks!